.syncpackrc

```diff
@@ -30,7 +30,27 @@
 			]
 		},
 		{
-			"dependencies": [ "**" ],
```

Comment:
Hey @rrennick, could you explain this change a bit more? It looks like we were previously ignoring syncpack on all packages from blocks. What was the aim of changing this to an explicit list? Was it to include certain packages?

If it's to include `@playwright/test`, which it looks to be, could we just add a version group higher up in the config to explicitly match that?

According to [the docs](https://jamiemason.github.io/syncpack/guide/version-groups/):

> Every instance of every dependency in your repo is assigned to the first version group it matches.

Reply:
>  It looks like we were previously ignoring syncpack on all packages from blocks. 

Yes, we were. That was added during the blocks merge because many of the blocks package versions are different. The intention was to allow us to continue to update packages in syncpack without breaking blocks.

> What was the aim of changing this to an explicit list? Was it to include certain packages?

The list includes all of the common existing packages in syncpack & the blocks repo. By excluding just these packages we can work on adding packages that currently aren't in syncpack and address packages that are in syncpack individually (eg. update/sync typescript & related packages).

Reply:
Okay, I just wasn't sure if it'd be cleaner to just add any packages that we'd want to manage for Blocks with syncpack above this line, as opposed to explicitly listing each wildcard, but either direction has the potential to cause confusion, so I'm happy with this approach. Thanks for the explanation.